packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -12,6 +12,11 @@
   "force_source_push_default_scratch_org_text": "SFDX: Push Source to Default Scratch Org",
   "force_source_push_force_default_scratch_org_text": "SFDX: Push Source to Default Scratch Org and Override Conflicts",
   "force_source_status_text": "SFDX: View All Changes (Local and in Default Scratch Org)",
+  "force_source_legacy_pull_default_scratch_org_text": "SFDX: Pull Source from Default Scratch Org (Legacy)",
```

Comment:
@sbudhirajadoc Can you please review the text for the new commands

Reply:
Also @AnanyaJha are there any user facing docs to add/update about the legacy/beta switchover? Or would the pinned issue you will be creating suffice?

Reply:
looks good to me.